### PR TITLE
feat: remove extra string from CustomMessage

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/AC-102 
I think people want us to display just what the server sends us.

### CODE REVIEW

File: src/components/CustomMessageBody.tsx [hunk 1]
```diff
@@ -24,28 +24,14 @@ const Text = styled.div`
...
 
-type SourceUrl = {
-  source_url: string;
-};
 
 export default function CustomMessageBody(props: Props) {
-  const { message, data } = props;
-  let extraStr = '';
-
-  if (data) {
```
Comment (sravan-s) [inline src/components/CustomMessageBody.tsx:38]: this was the info icon, positive we want to remove this?

**Review by sravan-s — APPROVED** [review]

File: src/components/CustomMessageBody.tsx (hunk 1 above)
Comment (sravan-s) [reply src/components/CustomMessageBody.tsx:38]: wait for liam too

File: src/components/CustomMessageBody.tsx [hunk 2]
```diff
@@ -24,28 +24,14 @@ const Text = styled.div`
...
-    const sourceUrl: string = obj['source_url'];
-    if (sourceUrl) {
-      const anchor = `<a href="${sourceUrl}" target="_blank">${sourceUrl}</a>`;
-      extraStr = ` I can answer your questions based on ${anchor}. Ask away!`;
-    }
-  }
+  const { message } = props;
 
   return <Root>
     {/* fixme: must sanitize message before dangerouslySettingInnerHTML*/}
```
Comment (sravan-s) [inline src/components/CustomMessageBody.tsx:34]: ps. @AhyoungRyu if you have time, can you apply sanitation here

File: src/components/CustomMessageBody.tsx (hunk 1 above)
Comment (AhyoungRyu) [reply src/components/CustomMessageBody.tsx:38]: Which is the part for parsing the icon? It seems there're only an anchor tag and "I can~" text.

Comment (sravan-s) [reply src/components/CustomMessageBody.tsx:38]: Ah, I misread the anchor~

**Pull request merged**